### PR TITLE
Add test for unicode characters in Python backend

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

**Review by deadeyegoodwin — DISMISSED** [review]

File: qa/python_models/string/model.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            input_tensors = request.inputs()
+            in_0 = pb_utils.get_input_tensor_by_name(request, "INPUT0")
+            unicode_character = '😀'
+            if str(in_0.as_numpy().tobytes(), encoding='utf-8') == unicode_character:
+                out_tensor_0 = pb_utils.Tensor("OUTPUT0", np.array(['success'], dtype=np.bytes_))
+            else:
+                out_tensor_0 = pb_utils.Tensor("OUTPUT0", np.array(['failure'], dtype=np.bytes_))
+            responses.append(
+                pb_utils.InferenceResponse([out_tensor_0]))
+        return responses
```
Comment (CoderHam) [inline qa/python_models/string/model.py:55]: Add new line after

File: qa/L0_backend_python/python_test.py [hunk 2]
```diff
@@ -245,6 +245,22 @@ def test_ensemble(self):
...
+            input_data = np.array([bytes(utf8, encoding='utf-8')], dtype=np.bytes_)
+            inputs = [
+                httpclient.InferInput("INPUT0", shape,
+                                      np_to_triton_dtype(input_data.dtype))
+            ]
+            inputs[0].set_data_from_numpy(input_data)
+            result = client.infer(model_name, inputs)
+            output0 = result.as_numpy('OUTPUT0')
+            self.assertTrue(output0 is not None)
+            self.assertTrue(str(output0[0], encoding='ascii') == 'success')
```
Comment (CoderHam) [inline qa/L0_backend_python/python_test.py:262]: Instead of returning success I would pass unicode and have an identity model that returns the same so you can verify that the handling of unicode for outputs is also validated.

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: qa/L0_backend_python/python_test.py (hunk 2 above)
Comment (Tabrizian) [reply qa/L0_backend_python/python_test.py]: Fixed. Although, since it is just a blob of bytes I don't think it makes a difference.

File: qa/python_models/string/model.py (hunk 1 above)
Comment (Tabrizian) [reply qa/python_models/string/model.py:51]: It does have a new line. Whenever it doesn't, it shows a little sign at the end of the line.

**Review by deadeyegoodwin — APPROVED** [review]
